airbnb_project/docker-compose.yml

```diff
@@ -1,22 +1,22 @@
-version: '3.8'
 services:
   db:
     image: postgres:16.4-bullseye
-    container_name: postgres_db
+    container_name: airbnb_postgres
     env_file:
       - ./.env
```

Comment:
i think it looks for `.env` in same folder?
maybe we should move both `.env` and `docker-compose.yml` to the root repo? just in `airbnb-regulation` folder?
sth like this:
```
airbnb-regulation/
├── airbnb_project/           # Django project directory
│   ├── manage.py
│   ├── airbnb_project/       # Django project settings and main app
│   ├── apps/                 # Other Django apps if any
│   └── ...
├── .env                      # Environment variables for Docker Compose and Django
├── docker-compose.yml        # Docker Compose configuration
├── Dockerfile                # Dockerfile to build the Django app container
├── pyproject.toml            # Poetry dependencies and configuration
├── README.md                 # Project documentation
└── other-files/
```

Reply:
Yeah this makes sense. As mentioned above, I'm gonna change things around after your pr is merged to use the single docker file you have created.

Reply:
Hey @immangat i just merged my pr so you can use the dockerfile